src/lib/y2network/autoinst_profile/routing_section.rb

```diff
@@ -80,6 +80,9 @@ def initialize(*_args)
       # @param hash [Hash] see {.new_from_hashes}
       def init_from_hashes(hash)
         super
+        ip_forward = hash["ip_forward"]
+        @ipv4_forward = hash["ipv4_forward"] || ip_forward
```

Comment:
You could do `@ipv4_forward = hash["ipv4_forward"] || hash["ip_forward"]` directly.